content/en/docs/setup/getting-started/test.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+
+# Download Istio
+# Skipping this as we use the istioctl built from istio/istio reference
+
+# Install Istio
+# @setup profile=none
+snip_install_istio_install_1
+_wait_for_deployment istio-system istiod
+
+# Label the namespace
+# remove the injection label to prevent the following command from failing
+kubectl label namespace default istio-injection-
+snip_install_istio_install_2
+
+# Cleanup sleep (not sure where it comes from)
+cleanup_sleep_sample
```

Comment:
```suggestion
# TODO: how to make sure previous test cleaned up everything?
set +e
kubectl delete pods -l app=sleep --force
set -e
```

Reply:
I thought that the problem was a sleep pod from a previous test was still in the terminating state, but it looks like there really is a bug in some other test that is not cleaning up its sleep service. I guess you should put back the call to cleanup_sleep_sample with a comment: TODO remove when cleanup bug in other test is fixed.